webapp/components/Form.tsx

```diff
@@ -135,127 +150,155 @@ const ReceiptForm = (): JSX.Element => {
 
         return (
           <form onSubmit={handleSubmit}>
-            <>
-              <div className={'grid grid-cols-1 gap-6 sm:grid-cols-2 my-2'}>
-                <FormInput
-                  name="name"
-                  label="Navn"
-                  required
-                  helperText="Ditt fulle navn, slik kvitteringen viser"
-                  autoFocus
-                />
-
-                <FormInput
-                  name="mailFrom"
-                  label="Din epost"
-                  required
-                  helperText="Et kopi av skjemaet vil bli sendt hit"
-                  validators={[emailValidator]}
-                />
+            <div
+              className={'grid grid-cols-1 gap-4 sm:grid-cols-2 sm:gap-6 my-2'}
+            >
+              <FormInput
+                name="name"
+                label="Navn"
+                required
+                helperText="Ditt fulle navn, slik kvitteringen viser"
+                autoFocus
+              />
 
-                <FormInput
-                  name="committee"
-                  label="Komité/gruppe"
-                  required
-                  helperText={'Den komiteen/gruppa som skylder deg penger'}
-                />
+              <FormInput
+                name="mailFrom"
+                label="Din e-post"
+                required
+                helperText="Et kopi av skjemaet vil bli sendt hit"
```

Comment:
```suggestion
                helperText="En kopi av skjemaet vil bli sendt hit"
```
I see this was not your fault